packages/core/platform/platform.js

```diff
@@ -48,8 +48,6 @@ const webOSVersion = {
 };
 
 const platforms = [
-	// Windows Phone 7 - 10
-	{platform: 'windowsPhone', regex: /Windows Phone (?:OS )?(\d+)[.\d]+/},
```

Comment:
Don't we need to put the deprecation before removing it?

Reply:
Modified to add deprecated notice to a change log first.